src/quantum/config.js

```diff
@@ -240,16 +240,20 @@ const PLATFORMS = [
     browser: 'fenix',
     label: 'Firefox Preview p2 (aarch64)',
     platform: 'fenix-p2-aarch64',
-    platformFilter: {
-      and: [
-        { eq: { platform: 'android-hw-p2-8-0-aarch64' } },
-        { eq: { platform: 'android-hw-p2-8-0-android-aarch64' } },
+    platformFilter: {and: [
+        {or: [
+          {eq: {
+            platform: 'android-hw-p2-8-0-aarch64'
+          }},
+          {eq: {
+            platform: 'android-hw-p2-8-0-android-aarch64'
+          }},
+        ]},
```

Comment:
Instead of 

        {or: [
          {eq: {
            platform: 'android-hw-p2-8-0-aarch64'
          }},
          {eq: {
            platform: 'android-hw-p2-8-0-android-aarch64'
          }},
        ]}

you can use 

          {in: {platform: [
              'android-hw-p2-8-0-aarch64', 
              'android-hw-p2-8-0-android-aarch64'
          ]}}

https://github.com/mozilla/ActiveData/blob/dev/docs/jx_expressions.md#in-operator

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/vendor/jx/expressions.js#L167...L173

I will add more comments